pkg/rpc/server/server.go

```diff
@@ -1557,16 +1559,52 @@ func (s *Server) getCommittee(_ request.Params) (interface{}, *response.Error) {
 
 // invokeFunction implements the `invokeFunction` RPC call.
 func (s *Server) invokeFunction(reqParams request.Params) (interface{}, *response.Error) {
+	tx, verbose, respErr := s.getInvokeFunctionParams(reqParams[1:])
+	if respErr != nil {
+		return nil, respErr
+	}
+	return s.runScriptInVM(trigger.Application, tx.Script, util.Uint160{}, tx, nil, verbose)
+}
+
+// invokeFunctionHistoric implements the `invokeFunctionHistoric` RPC call.
+func (s *Server) invokeFunctionHistoric(reqParams request.Params) (interface{}, *response.Error) {
+	if s.chain.GetConfig().KeepOnlyLatestState {
+		return nil, response.NewInvalidRequestError("'invokefunctionhistoric' is not supported", errKeepOnlyLatestState)
+	}
+	root, err := reqParams.Value(0).GetUint256()
```

Comment:
I'd accept both block number and stateroot hash.

Reply:
Maybe if we accept block number, then stateroot may be omitted?